roles/ssp/templates/config/config-1.17.php.j2

```diff
@@ -121,7 +121,7 @@ $config = [
      * metadata listing and diagnostics pages.
      * You can also put a hash here; run "bin/pwgen.php" to generate one.
      */
-    'auth.adminpassword' => '{{ ssp_adminpasswordhash }}',
+    'auth.adminpassword' => '{{ ssp_adminpassword | password_hash('ldap_salted_sha256', ssp_adminpassword) }}',
```

Comment:
I suggest that we introduce a new paramater, i.e. `ssp_adminpassword_salt`. Effectively this will become:
`'auth.adminpassword' => '{{ ssp_adminpassword | password_hash('ldap_salted_sha256', ssp_adminpassword+ssp_adminpassword_salt)  }}'`

`defaults/main.yml` needs to be updated to document how the secret salt can be generated similarly to how `ssp_secretsalt` is described in https://github.com/rciam/rciam-deploy/blob/0f3027614ee06cfaaf6fe16a1021f7109ad00a90/roles/ssp/defaults/main.yml#L24
The recommended command for generating the secret salt should take into account any size limit considerations